### ISSUE TITLE
Create Accessibility Behaviour for Dropdown component.

### ISSUE THREAD

**Issue by silviuaavram** [microsoft/fluent-ui-react#468]
## Feature Request
Original plan is to finish the Dropdown Component PR with the aria attributes directly added, as the PR is large enough for review. https://github.com/stardust-ui/react/pull/422

After that PR will be merged, will create another PR that will remove aria from component and add it to behaviour. Will close this issue once that PR will be merged

**Comment by levithomason** [comment]
I agree with this, however, we are using Downshift.  I am of the mind that we should replace downshift with Stardust concepts of state/style/accessibility abstractions.

How do you see these working together?

**Comment by levithomason** [comment]
@silviuavram  ^
